tests/unit/test-class-payment-information.php

```diff
@@ -17,6 +18,7 @@ class Payment_Information_Test extends WP_UnitTestCase {
 	const PAYMENT_METHOD_REQUEST_KEY = 'wcpay-payment-method';
 	const PAYMENT_METHOD             = 'pm_mock';
 	const TOKEN_REQUEST_KEY          = 'wc-' . \WC_Payment_Gateway_WCPay::GATEWAY_ID . '-payment-token';
```

Comment:
To maintain consistency, let's use `const CARD_TOKEN_REQUEST_KEY = 'wc-' . Card::GATEWAY_ID . '-payment-token';` here.